wpilibc/src/main/native/cpp/smartdashboard/MechanismObject2d.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "frc/smartdashboard/MechanismObject2d.h"
+#include <wpi/StringMap.h>
```

Comment:
No need to include StringMap.h here, since you include it in the MechanismObject2d.h header.